lib/rules/no-jquery-methods.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * @fileOverview Disallow the use of specific jQuery methods.
+ * @author Marc Lynch
+ */
+'use strict';
+
+/**
+ * Determines if this expression matches the method blacklist invocation.
+ * @param {ASTNode} node The identifier node.
+ * @returns {Boolean} Returns true if the expression matches, otherwise false.
+ */
+
+function get(node, path) {
```

Comment:
Is this duplicate code from our utils that was copypasta-ed into this when transporting form astexplorer? If so, can we pew pew?

🔫